ivy/functional/backends/torch/general.py

```diff
@@ -133,12 +134,15 @@ def to_numpy(
         x = x.resolve_neg().resolve_conj()
         if copy:
             if x.dtype is torch.bfloat16:
```

Comment:
actually we could simplify this to do something like this instead so that we don't need to cast `bfloat16` to `float32`, what do you think?
```
if copy:
    return np.array(x.tolist(), dtype=ivy.as_ivy_dtype(x.dtype))
```